app/models/rubric_criterion.rb

```diff
@@ -170,10 +180,6 @@ def to_yml
         'peer_visible' => self.peer_visible } }
   end
 
-  def weight
```

Comment:
Don't delete this method unless you've first checked that it's not used anywhere. Remember that in Ruby, it's possible to call a method without using parentheses, so look for a `<rubric criterion object>.weight`.